package.json

```diff
@@ -47,6 +48,7 @@
     "@types/react-dom": "^17",
     "antd": "^5.0.0",
     "apexcharts": "^3.40.0",
+    "clsx": "^2.0.0",
```

Comment:
Already bundled with MUI, but it is nice to use ourselves as well given that it's already there and not inflating the size.